recipe/meta.yaml

```diff
@@ -1,40 +1,48 @@
 {% set name = "Flask-Caching" %}
-{% set version = "1.10.1" %}
+{% set version = "2.0.1" %}
 
 package:
   name: "{{ name|lower }}"
   version: "{{ version }}"
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: cf19b722fcebc2ba03e4ae7c55b532ed53f0cbf683ce36fafe5e881789a01c00
+  sha256: 10df200a03f032af60077befe41779dd94898b67c82040d34e87210b71ba2638
 
 build:
   number: 0
-  noarch: python
   script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
+  skip: True # [py<37]
 
 requirements:
   host:
     - python
     - pip
+    - setuptools
+    - wheel
   run:
     - python
-    - flask
-    - werkzeug >=0.12
+    - cachelib >=0.9.0
+    - flask <3
+
 
 test:
   imports:
     - flask_caching
     - flask_caching.backends
+  requires:
+    - pip
+  commands:
+    - pip check
 
 about:
-  home: https://github.com/sh4nks/flask-caching
```

Comment:
`https://github.com/sh4nks/flask-caching` is the correct home URL, https://github.com/pallets-eco/flask-caching/blob/7609ac1636e7d8de6161fa3e1275de872a4362f5/setup.cfg#L4